<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>420</width>
    <height>369</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>421</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
	font: 14pt &quot;.AppleSystemUIFont&quot;;
  qproperty-alignment: 'AlignVCenter | AlignRight';
  border: 1px solid rgb(254, 248, 245);
color: rgb(85, 51, 79);
background-color : rgb(254, 248, 245);

}

background-color : white;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_ac">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>70</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>60</width>
      <height>60</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
border:1px solid rgb(254, 248, 245);
background-color: rgb(252, 241, 236);
color: rgb(133, 84, 117);;
}
QPushButton:pressed {
     background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 rgb(254, 248, 245), stop: 1 rgb(252, 241, 236));
}</string>
    </property>
    <property name="text">
     <string>AC</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_C">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>70</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>60</width>
      <height>60</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
border:1px solid rgb(254, 248, 245);
background-color: rgb(252, 241, 236);
color: rgb(133, 84, 117);;
}
QPushButton:pressed {
     background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 rgb(254, 248, 245), stop: 1 rgb(252, 241, 236));
}</string>
    </property>
    <property name="text">
     <string>C</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_mod">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>70</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>60</width>
      <height>60</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
border:1px solid rgb(254, 248, 245);
background-color: rgb(252, 241, 236);
color: rgb(133, 84, 117);;
}
QPushButton:pressed {
     background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 rgb(254, 248, 245), stop: 1 rgb(252, 241, 236));
}</string>
    </property>
    <property name="text">
     <string>mod</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_8">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>130</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>60</width>
      <height>60</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
border:1px solid rgb(254, 248, 245);
background-color: rgb(251, 230, 219);
color: rgb(133, 84, 117);;
}
QPushButton:pressed {
    
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 rgb(255, 238, 232), stop: 1 rgb(255, 232, 224));

}</string>
    </property>
    <property name="text">
     <string>8</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_7">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>130</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>60</width>
      <height>60</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
border:1px solid rgb(254, 248, 245);
background-color: rgb(251, 230, 219);
color: rgb(133, 84, 117);;
}
QPushButton:pressed {
    
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 rgb(255, 238, 232), stop: 1 rgb(255, 232, 224));

}</string>
    </property>
    <property name="text">
     <string>7</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_9">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>130</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>60</width>
      <height>60</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
border:1px solid rgb(254, 248, 245);
background-color: rgb(251, 230, 219);
color: rgb(133, 84, 117);;
}
QPushButton:pressed {
    
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 rgb(255, 238, 232), stop: 1 rgb(255, 232, 224));

}</string>
    </property>
    <property name="text">
     <string>9</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_5">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>190</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>60</width>
      <height>60</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
border:1px solid rgb(254, 248, 245);
background-color: rgb(251, 230, 219);
color: rgb(133, 84, 117);;
}
QPushButton:pressed {
    
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 rgb(255, 238, 232), stop: 1 rgb(255, 232, 224));

}</string>
    </property>
    <property name="text">
     <string>5</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>190</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>60</width>
      <height>60</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
border:1px solid rgb(254, 248, 245);
background-color: rgb(251, 230, 219);
color: rgb(133, 84, 117);;
}
QPushButton:pressed {
    
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 rgb(255, 238, 232), stop: 1 rgb(255, 232, 224));

}</string>
    </property>
    <property name="text">
     <string>4</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_6">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>190</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>60</width>
      <height>60</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
border:1px solid rgb(254, 248, 245);
background-color: rgb(251, 230, 219);
color: rgb(133, 84, 117);;
}
QPushButton:pressed {
    
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 rgb(255, 238, 232), stop: 1 rgb(255, 232, 224));

}</string>
    </property>
    <property name="text">
     <string>6</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>250</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>60</width>
      <height>60</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
border:1px solid rgb(254, 248, 245);
background-color: rgb(251, 230, 219);
color: rgb(133, 84, 117);;
}
QPushButton:pressed {
    
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 rgb(255, 238, 232), stop: 1 rgb(255, 232, 224));

}</string>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_1">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>250</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>60</width>
      <height>60</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
border:1px solid rgb(254, 248, 245);
background-color: rgb(251, 230, 219);
color: rgb(133, 84, 117);;
}
QPushButton:pressed {
    
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 rgb(255, 238, 232), stop: 1 rgb(255, 232, 224));

}</string>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>250</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>60</width>
      <height>60</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
border:1px solid rgb(254, 248, 245);
background-color: rgb(251, 230, 219);
color: rgb(133, 84, 117);
}
QPushButton:pressed {
    
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 rgb(255, 238, 232), stop: 1 rgb(255, 232, 224));

}</string>
    </property>
    <property name="text">
     <string>3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_dot">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>310</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>60</width>
      <height>60</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
border:1px solid rgb(254, 248, 245);
background-color: rgb(251, 230, 219);
color: rgb(133, 84, 117);;
}
QPushButton:pressed {
    
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 rgb(255, 238, 232), stop: 1 rgb(255, 232, 224));

}</string>
    </property>
    <property name="text">
     <string>.</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_0">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>310</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>60</width>
      <height>60</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
border:1px solid rgb(254, 248, 245);
background-color: rgb(251, 230, 219);
color: rgb(133, 84, 117);;
}
QPushButton:pressed {
    
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 rgb(255, 238, 232), stop: 1 rgb(255, 232, 224));

}</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_equal">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>310</y>
      <width>121</width>
      <height>60</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>60</width>
      <height>60</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
border:1px solid rgb(254, 248, 245);
background-color: rgb(130, 87, 122);
color: rgb(251, 230, 219);
}
QPushButton:pressed {
background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 rgb(145, 100, 137), stop: 1 rgb(130, 87, 122));
    
}</string>
    </property>
    <property name="text">
     <string>=</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_plus">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>250</y>
      <width>61</width>
      <height>60</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>60</width>
      <height>60</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
border:1px solid rgb(254, 248, 245);
background-color: rgb(252, 241, 236);
color: rgb(133, 84, 117);;
}
QPushButton:pressed {
     background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 rgb(254, 248, 245), stop: 1 rgb(252, 241, 236));
}</string>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_mult">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>130</y>
      <width>61</width>
      <height>60</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>60</width>
      <height>60</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
border:1px solid rgb(254, 248, 245);
background-color: rgb(252, 241, 236);
color: rgb(133, 84, 117);;
}
QPushButton:pressed {
     background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 rgb(254, 248, 245), stop: 1 rgb(252, 241, 236));
}</string>
    </property>
    <property name="text">
     <string>*</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_divide">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>70</y>
      <width>61</width>
      <height>60</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>60</width>
      <height>60</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
border:1px solid rgb(254, 248, 245);
background-color: rgb(252, 241, 236);
color: rgb(133, 84, 117);;
}
QPushButton:pressed {
     background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 rgb(254, 248, 245), stop: 1 rgb(252, 241, 236));
}</string>
    </property>
    <property name="text">
     <string>/</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_minus">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>190</y>
      <width>61</width>
      <height>60</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>60</width>
      <height>60</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
border:1px solid rgb(254, 248, 245);
background-color: rgb(252, 241, 236);
color: rgb(133, 84, 117);;
}
QPushButton:pressed {
     background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 rgb(254, 248, 245), stop: 1 rgb(252, 241, 236));
}</string>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_ln">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>250</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>60</width>
      <height>60</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
border:1px solid rgb(254, 248, 245);
background-color: rgb(252, 241, 236);
color: rgb(133, 84, 117);;
}
QPushButton:pressed {
     background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 rgb(254, 248, 245), stop: 1 rgb(252, 241, 236));
}</string>
    </property>
    <property name="text">
     <string>ln</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_asin">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>70</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>60</width>
      <height>60</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
border:1px solid rgb(254, 248, 245);
background-color: rgb(252, 241, 236);
color: rgb(133, 84, 117);;
}
QPushButton:pressed {
     background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 rgb(254, 248, 245), stop: 1 rgb(252, 241, 236));
}</string>
    </property>
    <property name="text">
     <string>asin</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_acos">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>130</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>60</width>
      <height>60</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
border:1px solid rgb(254, 248, 245);
background-color: rgb(252, 241, 236);
color: rgb(133, 84, 117);;
}
QPushButton:pressed {
     background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 rgb(254, 248, 245), stop: 1 rgb(252, 241, 236));
}</string>
    </property>
    <property name="text">
     <string>acos</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_atan">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>190</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>60</width>
      <height>60</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
border:1px solid rgb(254, 248, 245);
background-color: rgb(252, 241, 236);
color: rgb(133, 84, 117);;
}
QPushButton:pressed {
     background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 rgb(254, 248, 245), stop: 1 rgb(252, 241, 236));
}</string>
    </property>
    <property name="text">
     <string>atan</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_graph">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>310</y>
      <width>121</width>
      <height>60</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>60</width>
      <height>60</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
border:1px solid rgb(254, 248, 245);
background-color: rgb(252, 241, 236);
color: rgb(133, 84, 117);;
}
QPushButton:pressed {
     background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 rgb(254, 248, 245), stop: 1 rgb(252, 241, 236));
}</string>
    </property>
    <property name="text">
     <string>graph</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_cos">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>130</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>60</width>
      <height>60</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
border:1px solid rgb(254, 248, 245);
background-color: rgb(252, 241, 236);
color: rgb(133, 84, 117);;
}
QPushButton:pressed {
     background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 rgb(254, 248, 245), stop: 1 rgb(252, 241, 236));
}</string>
    </property>
    <property name="text">
     <string>cos</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_tan">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>190</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>60</width>
      <height>60</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
border:1px solid rgb(254, 248, 245);
background-color: rgb(252, 241, 236);
color: rgb(133, 84, 117);;
}
QPushButton:pressed {
     background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 rgb(254, 248, 245), stop: 1 rgb(252, 241, 236));
}</string>
    </property>
    <property name="text">
     <string>tan</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_sin">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>70</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>60</width>
      <height>60</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>420</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
border:1px solid rgb(254, 248, 245);
background-color: rgb(252, 241, 236);
color: rgb(133, 84, 117);;
}
QPushButton:pressed {
     background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 rgb(254, 248, 245), stop: 1 rgb(252, 241, 236));
}</string>
    </property>
    <property name="text">
     <string>sin</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_log">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>250</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>60</width>
      <height>60</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
border:1px solid rgb(254, 248, 245);
background-color: rgb(252, 241, 236);
color: rgb(133, 84, 117);;
}
QPushButton:pressed {
     background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 rgb(254, 248, 245), stop: 1 rgb(252, 241, 236));
}</string>
    </property>
    <property name="text">
     <string>log</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_x">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>310</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>60</width>
      <height>60</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
border:1px solid rgb(254, 248, 245);
background-color: rgb(252, 241, 236);
color: rgb(133, 84, 117);;
}
QPushButton:pressed {
     background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 rgb(254, 248, 245), stop: 1 rgb(252, 241, 236));
}</string>
    </property>
    <property name="text">
     <string>x</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_r_br">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>130</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>60</width>
      <height>60</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
border:1px solid rgb(254, 248, 245);
background-color: rgb(252, 241, 236);
color: rgb(133, 84, 117);;
}
QPushButton:pressed {
     background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 rgb(254, 248, 245), stop: 1 rgb(252, 241, 236));
}</string>
    </property>
    <property name="text">
     <string>)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_sqrt">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>190</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>60</width>
      <height>60</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
border:1px solid rgb(254, 248, 245);
background-color: rgb(252, 241, 236);
color: rgb(133, 84, 117);;
}
QPushButton:pressed {
     background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 rgb(254, 248, 245), stop: 1 rgb(252, 241, 236));
}</string>
    </property>
    <property name="text">
     <string>sqrt</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_l_br">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>70</y>
      <width>61</width>
      <height>60</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>60</width>
      <height>60</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
border:1px solid rgb(254, 248, 245);
background-color: rgb(252, 241, 236);
color: rgb(133, 84, 117);;
}
QPushButton:pressed {
     background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 rgb(254, 248, 245), stop: 1 rgb(252, 241, 236));
}</string>
    </property>
    <property name="text">
     <string>(</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_pow">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>250</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>60</width>
      <height>60</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
border:1px solid rgb(254, 248, 245);
background-color: rgb(252, 241, 236);
color: rgb(133, 84, 117);;
}
QPushButton:pressed {
     background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 rgb(254, 248, 245), stop: 1 rgb(252, 241, 236));
}</string>
    </property>
    <property name="text">
     <string>^</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
